### DISCUSSION TITLE
Privileges of organization members

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1621] [Team Posts]
Currently privileges of members in the GitHub organization sympa-community is as below.
![image](https://user-images.githubusercontent.com/5743546/106419929-ef748a00-649c-11eb-9b9c-ef8e10181422.png)

I suggest @xavierba to be an owner.  It's surprising for me that he's not the owner, given his activity.  See [documentation](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization) about details of privileges.

**Comment by racke** [comment]
I agree about @xavierba. On the other hand I have no idea why @Arnaud Grausem is in owner. I suggest to demote him to a member unless someone has good arguments against that.

**Comment by ikedas** [comment]
@racke, you links to a different person than @agrausem .

**Comment by racke** [comment]
Sorry, thanks for the correction.

**Comment by ldidry** [comment]
Ok for @xavierba, no opinion for @agrausem.

**Comment by dverdin** [comment]
OK for @xavierba . I don't even remember who is @agrausem ...
I'd be in favour of not keeping him as owner.

**Comment by ikedas** [comment]
Ownership
* To avoid unforeseen accidents, people who are considered to be unaware of their ownership may be removed.  I suppot removing @agrausem from owner.

Invitation
* We may feel free to invite new members by each.   Is there any objection?

Others
* Besides, as L Bo @elbeho has not been added to everyone (a group of discussion), I added.

**Comment by racke** [comment]
- Let's remove owner role from @agrausem
- Inviting members is fine, but please give a short introduction about the new member here first

**Comment by ikedas** [comment]
- @agrausem was removed from owner, and will remain a member.
  - @xavierba was added to owner.
